src/__experimental__/components/search/search.spec.js

```diff
@@ -472,4 +476,48 @@ describe("Search", () => {
       expect(input.prop("iconTabIndex")).toEqual(-1);
     });
   });
+
+  describe("when typing into the input", () => {
+    const stopPropagationFn = jest.fn();
+
+    afterEach(() => {
+      jest.clearAllMocks();
+    });
+
+    it("should stop propagation of the event for character keys", () => {
```

Comment:
I think that we can create a `it.each` for these examples

Reply:
I'm not sure there's any need for that here really, we only have 2 tests making the same assertion, the third test is asserting the opposite

Reply:
You are right! I thought 3 of them are about the same :)
Ignore my comment then